libobs/obs-video.c

```diff
@@ -322,6 +382,12 @@ static void render_convert_texture(struct obs_core_video *video,
 	gs_enable_blending(false);
 
 	if (video->convert_textures[0]) {
+
+		// Added for tracing direct ARGB encoding
+		blog(LOG_INFO,
+		     "=== [obs-video] render_convert_texture, from %p to %p",
```

Comment:
This appears to be called on every single frame, which creates a huge amount of log spam.  This should either be LOG_DEBUG, or ifdef'd out when not needed, or not included at all in a final merge.